CC21.9PullReq

```diff
@@ -0,0 +1 @@
+
```

Comment:
Most of these changes will require managing better meta data.  
I would suggest we expand on the [OriginFile](https://github.com/PrimeSonic/PrimeSonicSubnauticaMods/blob/master/CustomCraftSML/Serialization/OriginFile.cs) class, which is already linking a CC2 entry back to the file it came from.  

We might also include a new class specifically to tie together multiple files (both text and image) so that everything can be collected into a package before being patched in.

Reply:
Out of scope. This will be a larger feature is a potential future "CustomCraft3" mod.